kvm.yaml

```diff
@@ -1,3 +1,40 @@
+- version: 11.3.0
+  active: true
+  date: 2020-03-02T12:00:00Z
+  apps:
+    - app: cert-exporter
+      version: 1.2.1
+    - app: chart-operator
+      version: 0.11.4
+    - app: coredns
+      componentVersion: 1.6.5
+      version: 1.1.3
+    - app: kube-state-metrics
+      componentVersion: 1.9.2
+      version: 1.0.2
+    - app: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - app: net-exporter
+      version: 1.6.0
+    - app: nginx-ingress-controller
+      componentVersion: 0.29.0
+      version: 1.5.0
+    - app: node-exporter
+      componentVersion: 0.18.1
+      version: 1.2.0
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      provider: kvm
+      version: 0.23.1
+    - name: flannel-operator
+      version: 0.2.0
+    - name: kvm-operator
+      version: 3.11.0
```

Comment:
This should be right and the next tag we are suppose to create under kvm-operator